real_check/CMakeLists.txt

```diff
@@ -4,3 +4,5 @@
 if(${run_unittests})
     add_subdirectory(test)
 endif()
+
+add_custom_target(realcheck SOURCES real_check.ps1)
```

Comment:
Besides having this target in Visual Studio, is there any other reason to have this line here?

Reply:
Also the script should only run on Windows (presumably Linux would produce the same results).

So somewhere there should be some "if LINUX... bye-bye".

Reply:
I added the target so it would rerun real_check if the script was modified. No other reason.

The check is turned off by default. 

I was thinking we can achieve the OS-specific running of the check by only turning on the 'run_realscheck' option in windows builds in the pipeline. Would we still need a OS check in CMakeLists?

Reply:
Adding check for platform in next iteration.